selfdrive/controls/controlsd.py

```diff
@@ -162,8 +162,8 @@ def __init__(self, sm=None, pm=None, can_sock=None, CI=None):
     self.active = False
     self.can_rcv_error = False
     self.soft_disable_timer = 0
-    self.v_cruise_kph = 255
-    self.v_cruise_cluster_kph = 255
+    self.v_cruise_kph = 0
+    self.v_cruise_cluster_kph = 0
```

Comment:
Need to make sure all the carcontrollers set the correct values when we haven't got a set speed yet, then should be good to merge.